.github/workflows/backup-restore-test.yml

```
@@ -0,0 +1,183 @@
+  PG_CLIENT_VERSION: "16"
+
+concurrency:
+  group: backup-restore-test-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  restore-test:
+    name: Restore Test
+    # Disabled by default until RESTORE_TARGET_DB_URL is configured in GitHub Secrets.
+    # Enable by setting the secret and removing this conditional.
+    if: ${{ secrets.RESTORE_TARGET_DB_URL != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)